### PR TITLE
Chapter 8: Enrollment stats

### PR DESCRIPTION
**Pull request by dgilboa**
Add solution (incl tests).

### CODE REVIEW

File: realpython/ch08/enrollment_stats_test.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2015 Lightricks. All rights reserved.
+import unittest
+from enrollment_stats import median, mean, enrollment_stats, Uni
+
+
+class TestEnrollmentStats(unittest.TestCase):
+
+    def test_mean(self):
```
Comment (dlahyani) [inline realpython/ch08/enrollment_stats_test.py]: For future PRs please make sure every test tests exactly one thing, usually one scenario of input and output validation. It makes it easier to read and to understand why the test has failed. In this case if one scenario fails the others will simply not run, in other case one scenario may change the state of the class-under-test and affect the following scenarios you are testing.

File: realpython/ch08/enrollment_stats_test.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2015 Lightricks. All rights reserved.
+import unittest
+from enrollment_stats import median, mean, enrollment_stats, Uni
```
[the remark this answers is not in the document]
Comment (dgilboa) [reply realpython/ch08/enrollment_stats_test.py]: really weird but I'm sure it can be set automatically in the pycharm but I can't see where - any idea ?

**Comment by dgilboa** [comment]
@dlahyani - Fixed by the comments for your review.

File: realpython/ch08/enrollment_stats_test.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dlahyani) [reply realpython/ch08/enrollment_stats_test.py]: AFAIK PyCharm does not support such separation. If you find a way to do it let us know.

**Pull request merged**